.gitlab-ci.yml

```diff
@@ -748,10 +793,24 @@ build/dpcpp/dgpu/debug/shared:
     C_COMPILER: "gcc"
     CXX_COMPILER: "dpcpp"
     BUILD_DPCPP: "ON"
-    BUILD_TYPE: "Debug"
+    BUILD_TYPE: "Release"
     DPCPP_SINGLE_MODE: "ON"
     SYCL_DEVICE_FILTER: "Level_Zero:GPU"
```

Comment:
Does this work, or do you need to use the device_type as well?

Reply:
yes for the result.
but only Level_Zero has effect.
the GPU is the only one which has level_zero backend, so it gives the same result